src/main/java/frc/robot/constants/RobotMap.java

```diff
@@ -29,6 +29,11 @@
             public static final int REAR_RIGHT_ANGLE_MOTOR_ID = 5;
             public static final int REAR_RIGHT_ANGLE_ENCODER_ID = 10;
         }
+
+        public static class Shooter {
+            public static final int RIGHT_MOTOR_ID = 2;
+            public static final int LEFT_MOTOR_ID = 3;
```

Comment:
left above right just because of the alphabet

Reply:
Refer to above

Reply:
fix